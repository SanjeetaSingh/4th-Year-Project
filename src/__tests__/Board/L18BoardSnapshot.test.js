import renderer from 'react-test-renderer'

import Level18 from '../../Level18/L18-Board/L18-Board'

it('snapshot test level 18', () => {
    const tree =  renderer.create(<Level18 />).toJSON()
    expect(tree).toMatchSnapshot();
});