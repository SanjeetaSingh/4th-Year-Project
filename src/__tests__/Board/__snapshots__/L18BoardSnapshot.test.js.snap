// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test level 18 1`] = `
<div
  className="container"
>
  <div
    className="boardcontainer2"
  >
    <div
      className="board16"
    >
      <div
        className="box"
        data-col={0}
        data-row={0}
      />
      <div
        className="box"
        data-col={1}
        data-row={0}
      />
      <div
        className="box"
        data-col={2}
        data-row={0}
      >
        <img
          alt="d"
          class="goal"
          id="cat"
          src="assets/cat.png"
        />
      </div>
      <div
        className="box"
        data-col={3}
        data-row={0}
      />
      <div
        className="box"
        data-col={4}
        data-row={0}
      />
      <div
        className="box"
        data-col={0}
        data-row={1}
      />
      <div
        className="box"
        data-col={1}
        data-row={1}
      >
        <img
          alt="d"
          class="goal"
          id="food"
          src="assets/bone.png"
        />
      </div>
      <div
        className="box"
        data-col={2}
        data-row={1}
      />
      <div
        className="box"
        data-col={3}
        data-row={1}
      />
      <div
        className="box"
        data-col={4}
        data-row={1}
      />
      <div
        className="box"
        data-col={0}
        data-row={2}
      />
      <div
        className="box"
        data-col={1}
        data-row={2}
      />
      <div
        className="box"
        data-col={2}
        data-row={2}
      />
      <div
        className="box"
        data-col={3}
        data-row={2}
      />
      <div
        className="box"
        data-col={4}
        data-row={2}
      />
      <div
        className="box"
        data-col={0}
        data-row={3}
      />
      <div
        className="box"
        data-col={1}
        data-row={3}
      />
      <div
        className="box"
        data-col={2}
        data-row={3}
      />
      <div
        className="box"
        data-col={3}
        data-row={3}
      />
      <div
        className="box"
        data-col={4}
        data-row={3}
      />
      <div
        className="box"
        data-col={0}
        data-row={4}
      />
      <div
        className="box"
        data-col={1}
        data-row={4}
      />
      <div
        className="box"
        data-col={2}
        data-row={4}
      />
      <div
        className="box"
        data-col={3}
        data-row={4}
      />
      <div
        className="box"
        data-col={4}
        data-row={4}
      >
        <img
          alt="d"
          class="image"
          id="dog"
          src="assets/dog.png"
        />
      </div>
    </div>
  </div>
  <div
    className="bottom-panel"
  >
    <div
      class="border"
    >
      <div
        class="level13Contain"
      >
        <h2>
          Level 18:
        </h2>
        <div
          class="speech16"
        >
          Help the dog get to his bone! Use the for loop again the variable steps will be a counter that will be passed into the for loop to determine how many time you would like to iterate through your commands. The count starts at 0 so be careful of that! The boolean need to check if the moving cat is present or not, the boolean will be used in the if statement which will get rid of the cat by barking. Use the same amount or less amount of commands mentioned in the top right corner.
          <br />
          <p
            class="hints"
          >
            Click on the information buttons to learn more about for, understand the for loop condition, how variables work and about if statments
            <br />
             Good luck! 
            <br />
             
          </p>
        </div>
        <div
          class="borderPanel"
        >
          <div
            class="toppart16"
          >
            <div
              class="containing"
            >
              <p
                class="number"
                data-testid="counter"
                id="count"
              >
                0/5
              </p>
            </div>
            <p
              class="titles"
            >
              Enter Sequence:
            </p>
            <div
              class="buttons-wrapper12"
            >
              <button
                class="seemingly-inner-button12"
                disabled={false}
                onClick={[Function]}
              >
                <i
                  class="fa fa-info"
                />
                    Variables
              </button>
              <p
                class="subtitles"
              >
                Variable Declaration
              </p>
            </div>
            <form
              class="var"
            >
              boolean catPresent =  
              <input
                class="input"
                id="var1"
                onChange={[Function]}
                size="5"
              />
                ;     
              <br />
              <br />
                int steps =  
              <input
                class="input"
                id="var3"
                onChange={[Function]}
                size="5"
              />
                ;
              <br />
              <br />
            </form>
            <p
              class="lists"
              id="action"
            />
          </div>
        </div>
        <div
          class="outside13"
        >
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Left
          </button>
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Up
          </button>
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Down
          </button>
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Right
          </button>
          <div
            class="buttons-wrapper5"
          >
            <button
              class="seemingly-inner-button"
              disabled={false}
              onClick={[Function]}
            >
              <i
                class="fa fa-info"
              />
            </button>
            <button
              class="button"
              disabled={false}
              onClick={[Function]}
              type="submit"
            >
              For
            </button>
          </div>
          <div
            class="buttons-wrapper5"
          >
            <button
              class="seemingly-inner-button"
              disabled={false}
              onClick={[Function]}
            >
              <i
                class="fa fa-info"
              />
            </button>
            <button
              class="button"
              disabled={false}
              onClick={[Function]}
              type="submit"
            >
              If Statement
            </button>
          </div>
          <button
            class="button"
            onClick={[Function]}
            type="submit"
          >
            Bark
          </button>
          <button
            class="buttonClear"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Clear
          </button>
          <button
            class="buttonSub"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
             Submit
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
